src/pyqasm/maps/gates.py

```diff
@@ -1111,6 +1111,41 @@ def two_qubit_gate_op(
     BasisSet.CLIFFORD_T: {"h", "t", "s", "cx", "tdg", "sdg"},
 }
 
+GATE_ENTITY_DATA = {
```

Comment:
Why have we chosen `ENTITY` here? I think we can also use something like `GATE_OPT_DATA` (short for OPTIMIZATION)